src/components/layout/marqueeBanner.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+import logo from "@public/images/logos/xplorer_logo.svg";
+
+const Divider = () => {
+  return (
+    <span className="block h-2 w-2 shrink-0 rounded-full bg-slate-800"></span>
+  );
+};
+
+const Banner = () => {
+  return (
+    <>
+      <Image src={logo} alt="logo" className="w-24" />
+      <Divider />
+      <p>Learn</p>
+      <Divider />
+      <p>Share</p>
+      <Divider />
+      <p>Grow</p>
+      <Divider />
+    </>
+  );
+};
+
+const MarqueeBanner = () => {
+  return (
+    <div className="flex h-[50px] items-center gap-10 overflow-hidden bg-primary-50/15">
+      <div className="marquee flex w-auto shrink-0 items-center justify-evenly gap-10 text-xl xl:w-full">
+        <Banner />
+        <Banner />
+      </div>
+      <div
+        className="marquee flex w-auto shrink-0 items-center justify-evenly gap-10 text-xl xl:w-full"
+        aria-hidden="true"
+      >
+        <Banner />
+        <Banner />
+      </div>
```

Comment:
just trying to understand: any reason for aria-hidden true in one of them?
and would refactoring them into a component (like MarqueeContent) benefit?

Reply:
It's for accessibility reasons. Since we repeat the same component multiple times, I thought it would be good to hide it from screen readers. But looking at it again, I think there is another flaw in this component. We already have `<Banner />` component repeating multiple times. So if we optimise for accessibiilty, it should be done on the `<Banner />` component. What do you say @sonam-serchan?

Reply:
Yeah sounds good!